### ISSUE TITLE
Can't connect to any servers.

### ISSUE THREAD

**Issue by DanyTPG** [mysteriumnetwork/node#2020]
Hi there. 
I tried using the mobile app but I cannot connect to any servers.
I am from Iran and I think your network might have been blocked by Iran's firewall as I cannot open your website without a VPN either.
Could you investigate this issue?

**Comment by zolia** [comment]
Hello, 

We also suspect that Iran is performing active blocking of MN. This can be limited  to discovery / broker elements only, but since we don't have any more information regarding Iran fw we cannot proceed in fixing it. 

If you are getting a list of servers on a mobile, that most likely mean that discovery works. Actual protocols of OpenVPN and Wireguard might be blocked. They can also be actively collecting our list of node IPs that providing services and blocking just them.

Does UDP protocol is allowed through your country's FW? All MN vpns work through that protocol.

**Comment by DanyTPG** [comment]
Hi again. 
On broadband internet I can load the server list but on mobile data server list doesn't load.
Also right now as I am writing this I am connected through OpenVPN UDP protocol hosted on my VPS server which I also run an MN node on. so I believe this will confirm neither OpenVPN nor UDP are blocked by Iran's firewall. Also if they actively collect MN nodes IP addresses and blocked them I wouldn't have been able to access my server.

**Comment by zolia** [comment]
>Also right now as I am writing this I am connected through OpenVPN UDP protocol hosted on my VPS server which I also run an MN node on

Where this VPS is located at? If its in Iran, most likely that is why it works. Maybe they apply protocol filtering only on a "border" gateways. 

Also, what mobile app version are you using? Version should be visible on a bottom of a screen on feedback page.

**Comment by DanyTPG** [comment]
> Where this VPS is located at? If its in Iran, most likely that is why it works. Maybe they apply protocol filtering only on a "border" gateways.
> 
> Also, what mobile app version are you using? Version should be visible on a bottom of a screen on feedback page.

No, my VPS is hosted in France. I also can't connect to my node which is this VPS.
The mobile app version is 0.22.5.107052.
I tried many of the servers and there are a very few of them which I can connect to.
Also submitted the mobile feedback

**Comment by zolia** [comment]
would you be able to connect consistently to 
 this node: 0xe5e54c844d8e47f4eb43b58b286ce89e18b4283c
Wait for up to 15 sec. to connect. To openvpn and wireguard.

Also, you can write to me on our discord directly. Is it available in Iran?

**Comment by DanyTPG** [comment]
Sure

**Comment by chompomonim** [comment]
@zolia is this issue related o ISP blocking UDP traffic?

**Comment by chompomonim** [comment]
Old issue, most probably is not relevant anymore. If you still have similar issue with newest versions of application, please open new issue.
